spec.md

```diff
@@ -221,6 +221,11 @@ that contains both context and data).
 * Constraints:
   * OPTIONAL
 
+### log-level
+* Type: String
+* Description:  The level of logging that the publisher may want to log this
```

Comment:
need to remove the 'may'.
Perhaps: The level of logging associated with this specific event (e.g. debug)

Can you also add a "Constraints" section? Probably with OPTIONAL